app/src/main/java/com/android/bookswap/model/map/BookManagerViewModel.kt

```diff
@@ -113,7 +114,7 @@ class BookManagerViewModel(
         launch {
           booksRepository.getBook { books ->
             if (books.isSuccess) {
-              _allBooks.value = books.getOrThrow()
+              _allBooks.value = books.getOrThrow().filter { it.userId != currentUserUUID }
```

Comment:
This is redundant with the line 168. The books with no users are filtered out, as the current user is removed from the allUser list, the books are removed at line 168